DashKit.xcodeproj/project.pbxproj

```diff
@@ -971,73 +909,6 @@
 			};
 			name = Release;
 		};
-		C16DA85422E93387008624C2 /* Debug */ = {
```

Comment:
Is this just the plugin for the watch that you are removing?

Reply:
Yes, it was some crud leftover from early watch plugin testing.